snuba/datasets/transactions_processor.py

```diff
@@ -77,7 +77,7 @@ def process_message(self, message, metadata=None) -> Optional[ProcessedMessage]:
             processed["trace_id"] = str(uuid.UUID(trace_id))
             processed["span_id"] = int(transaction_ctx["span_id"], 16)
             processed["transaction_op"] = _unicodify(transaction_ctx.get("op", ""))
-            processed["transaction_name"] = _unicodify(data["transaction"])
+            processed["transaction_name"] = _unicodify(data.get("transaction", ""))
```

Comment:
Accessing `data["transaction"]` if it's not there throws an exception and we bail here https://github.com/getsentry/snuba/pull/906/files#diff-7778db8248b2fbd2db8fa2a2e8ffeb22R99